deployment/frozen/testnet.toml

```diff
@@ -0,0 +1,20 @@
+[cluster]
+code_hash_list = [
+    # version v1 (no commit)
+    "0x598d793defef36e2eeba54a9b45130e4ca92822e1d193671f490950c3b856080",
+]
+
+[spore]
+code_hash_list = [
+    # version v1 (no commit)
+    "0xbbad126377d45f90a8ee120da988a2d7332c78ba8fd679aab478a19d6c133494",
+]
```

Comment:
## commit: [457e32d](https://github.com/sporeprotocol/spore-contract/commits/457e32d) (release)

- code_hash of contact spore              : 0xbbad126377d45f90a8ee120da988a2d7332c78ba8fd679aab478a19d6c133494
- code_hash of contact cluster            : 0x598d793defef36e2eeba54a9b45130e4ca92822e1d193671f490950c3b856080
- code_hash of contact cluster_agent      : 0xbdeadaf1fdf577b7e53a565fa8d0bba8986e504f0126a21841926ee6d4adac82
- code_hash of contact cluster_proxy      : 0xa2e60304c80b1a400499750fda098a8a0f47a85fbbb2561452782abcd84ea4dc

Reply:
choose this one, recorded